examples/example_config_files/ADLChargeDriftModel/drift_velocity_config_powerlaw.yaml

```diff
@@ -1,37 +0,0 @@
-model: ADLChargeDriftModel
-phi110: 1.25664
-drift:
-  velocity:
-    model: Bruyneel2006
-    temperature: 78
-    parameters:
-      e100:
-        mu0: 3.8609
-        beta: 0.805
-        E0: 51100
-        mun: -0.0171
-      e111:
-        mu0: 3.8536
-        beta: 0.641
-        E0: 53800
-        mun: 0.051
-      h100:
-        mu0: 6.1824
-        beta: 0.942
-        E0: 18500
-      h111:
-        mu0: 6.1215
-        beta: 0.662
-        E0: 18200
-temperature_dependence:
-  model: PowerLaw
```

Comment:
I'm worried that we're using the same name `"PowerLaw"` in the config files for the new model.

We could use `Omar1987`, in accordance to `Bruyneel2016` or `Bruyneel2006`.